lib/rbUtil.js

```diff
@@ -173,6 +173,23 @@ rbUtil.parseETag = function(etag) {
     }
 };
 
+/**
+ * Copies forwarded headers from restbase to request.
+ * If the same header was already set it takes precedence over
+ * the forwarded header.
+ */
```

Comment:
I am wondering if we should handle this transparently in restbase.js, similar to the way we systematically forward the request id. However, we should only forward these headers //Iff// access is restricted. If it isn't, then we should absolutely make sure that we don't pass any cookies along that could result in us getting access to privileged information.